lib/core/src/Cardano/Wallet/Primitive/CoinSelection/MA/RoundRobin.hs

```diff
@@ -116,6 +119,9 @@ runSelection available minimumBalance =
         , leftover = available
         }
 
+    extraSource :: Natural
+    extraSource = maybe 0 (fromIntegral . unCoin) mExtraSource
```

Comment:
**Suggestion:** Perhaps this could be nested in a `where` clause under the definition of `coinSelectionLens`?

Alternatively, we could call this `extraCoinSource`, to make it clear that it's a source of `Coin`, rather than some other asset.

Reply:
Sounds good :+1: